test/basic-querying.test.js

```diff
@@ -6,6 +6,7 @@
 // This test written in mocha+should.js
 'use strict';
 
+/* global getSchema:false, connectorCapabilities:false */
```

Comment:
We should fix (ie. not use) these globals at some point. ;) That is outside the scope of this PR though.